bcgov/cas-registration
# multiselect required

#1602 

To test this:
- Go to the first page of the `Register an Operation` form - `Operation Information`
- Select `Reporting Operation` in the Registration Purpose dropdown
- The `Regulated Product Name(s)` field will appear which is a `MultiSelectWidget`. Click `Save and Continue` to trigger validation.
<img width="1032" alt="Screenshot 2024-09-12 at 8 19 29 AM" src="https://github.com/user-attachments/assets/24f32e7f-3fa1-4b72-92d7-e26a3681579b">
